core/src/main/java/com/graphhopper/util/TranslationMap.java

```diff
@@ -35,7 +35,7 @@ public class TranslationMap {
     private static final List<String> LOCALES = Arrays.asList("ar", "ast", "bg", "bn_BN", "ca",
             "cs_CZ", "da_DK", "de_DE", "el", "eo", "es", "en_US", "fa", "fil", "fi",
             "fr_FR", "fr_CH", "gl", "he", "hr_HR", "hsb", "hu_HU", "in_ID", "it", "ja", "ko", "lt_LT", "ne",
-            "nl", "pl_PL", "pt_BR", "pt_PT", "ro", "ru", "sk", "sl_SI", "sr_RS", "sv_SE", "tr", "uk",
+            "nl", "nb_NO", "pl_PL", "pt_BR", "pt_PT", "ro", "ru", "sk", "sl_SI", "sr_RS", "sv_SE", "tr", "uk",
```

Comment:
"Lexicographical order"?

Reply:
Lexigraphicol order with respect to the country.